docs/index.html

```diff
@@ -4534,7 +4534,7 @@ <h3 id="_属性の優先順位_高いものから列挙"><a class="anchor" href=
 <tr>
 <td class="tableblock halign-left valign-top"><p class="tableblock"><code>asciidoctor</code></p></td>
 <td class="tableblock halign-left valign-top"><div><div class="paragraph">
-<p>プロセッサーがAsciidoctorの場合に設定する。</p>
+<p>プトセッサーがAsciidoctorの場合に設定する。</p>

```

Comment:
This line is not my fix but `./gradlew asciidoctor` generates this results according to the other people's work.